.github/workflows/publish.yml

```diff
@@ -1,29 +1,86 @@
-name: brew pr-pull
+name: Publish Bottles
 on:
-  pull_request_target:
-    types:
-      - labeled
+  pull_request:
+    paths:
+      - Formula/*.rb
+
 jobs:
-  pr-pull:
-    if: contains(github.event.pull_request.labels.*.name, 'pr-pull')
-    runs-on: ubuntu-22.04
+  build-bottles:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Set up Homebrew
         uses: Homebrew/actions/setup-homebrew@master
 
+      - run: brew test-bot --only-cleanup-before
+
+      - run: brew test-bot --only-setup
+
+      - run: brew test-bot --only-tap-syntax
+
+      - run: brew test-bot --only-formulae
+
+      - name: Upload bottles as artifact
+        uses: actions/upload-artifact@main
+        with:
+          name: bottles
+          path: '*.bottle.*'
+
+  brew-bottle:
+    runs-on: ubuntu-latest
+    needs: build-bottles
+    steps:
+      - name: Set up Homebrew
+        run: |
+          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+
+          echo "$(brew --prefix)/bin" >> $GITHUB_PATH
+
+          HOMEBREW_TAP_REPOSITORY="$(brew --repo "$GITHUB_REPOSITORY")"
+          if [[ -d "$HOMEBREW_TAP_REPOSITORY" ]]; then
+              rm -rf "$HOMEBREW_TAP_REPOSITORY"
+          fi
+          mkdir -vp "$HOMEBREW_TAP_REPOSITORY"
+
+          rm -rf "$GITHUB_WORKSPACE"
+          ln -vs "$HOMEBREW_TAP_REPOSITORY" "$GITHUB_WORKSPACE"
+
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
       - name: Set up git
         uses: Homebrew/actions/git-user-config@master
```

Comment:
[This action](https://github.com/Homebrew/actions/tree/master/git-user-config) defaults to `$GITHUB_ACTOR`...which for the example run was you:

```
Run Homebrew/actions/git-user-config@master
  with:
    token: ***
    username: kylewillmon

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/[2](https://github.com/phylum-dev/homebrew-cli/actions/runs/3431204306/jobs/5719536719#step:4:2)022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
/usr/bin/git config --global user.name Kyle Willmon
/usr/bin/git config --global user.email [6](https://github.com/phylum-dev/homebrew-cli/actions/runs/3431204306/jobs/5719536719#step:4:7)1606[7](https://github.com/phylum-dev/homebrew-cli/actions/runs/3431204306/jobs/5719536719#step:4:8)+kylewillmon@users.noreply.github.com
```

Is this desired...or should it be a bot account?

Reply:
> Is this desired...or should it be a bot account?

You tell me... I couldn't decide... But it probably doesn't matter because the commit will be squashed when the PR is merged.

Reply:
It should be fine to keep it as-is for now and then change it if a better reason emerges.